app/scripts/lib/stores/read-only-network-store.test.ts

```diff
@@ -0,0 +1,141 @@
+import log from 'loglevel';
+import nock from 'nock';
+import ReadOnlyNetworkStore from './read-only-network-store';
+import { MetaMaskStorageStructure } from './base-store';
+
+const FIXTURE_SERVER_HOST = 'localhost';
+const FIXTURE_SERVER_PORT = 12345;
+const FIXTURE_SERVER_ORIGIN = `http://${FIXTURE_SERVER_HOST}:${FIXTURE_SERVER_PORT}`;
+const FIXTURE_SERVER_PATH = '/state.json';
+
+const DEFAULT_INITIAL_STATE = {
+  data: { config: {} },
+};
+
+const MOCK_STATE = { data: { config: { foo: 'bar' } }, meta: { version: 1 } };
+
+/**
+ * Initiatilizes a ReadOnlyNetworkStore for testing
+ *
+ * @returns store - a ReadOnlyNetworkStore
+ */
+function setupReadOnlyNetworkStore() {
+  const store = new ReadOnlyNetworkStore();
+  return store;
+}
+
+/**
+ * Create a Nock scope for the fixture server response.
+ *
+ * @returns A Nock interceptor for the fixture server response.
+ */
+function mockFixtureServerInterceptor(): nock.Interceptor {
+  return nock(FIXTURE_SERVER_ORIGIN).get(FIXTURE_SERVER_PATH);
+}
+
+/**
+ * Create a Nock scope for the fixture server response, which will have a successful reply.
+ *
+ * @param state
+ */
+function setMockFixtureServerReply(
+  state: Record<string, unknown> = DEFAULT_INITIAL_STATE,
+): void {
+  mockFixtureServerInterceptor().reply(200, state);
+}
+
+describe('ReadOnlyNetworkStore', () => {
+  beforeEach(() => {
+    jest.resetModules();
+    nock.cleanAll();
+  });
+
+  describe('constructor', () => {
+    it('makes a request to the fixture server upon construction', async () => {
```

Comment:
This test doesn't seem that useful 🤔 It tests that a request is made, but we don't care about that. The job of this class is not to "make a network request", it's to load state from an external fixture service. The later tests already do a better job of testing this.

Reply:
removed in 1dd96eea6f